src/core/opamSystem.ml

```diff
@@ -585,8 +585,8 @@ let make_command
   let verbose =
     OpamStd.Option.default OpamCoreConfig.(!r.verbose_level >= 2) verbose
   in
-  (* Check that the command doesn't contain whitespaces *)
-  if None <> try Some (String.index cmd ' ') with Not_found -> None then
+  (* Check that the command doesn't contain whitespaces, except on Windows *)
+  if not Sys.win32 && String.contains ' ' cmd
     OpamConsole.warning "Command %S contains space characters" cmd;
```

Comment:
why do we still have this check at all? Is there something broken on unix?

Reply:
It's from this [comment](https://github.com/ocaml/opam/issues/5163#issuecomment-1178703600), that advice to be conservative.

Reply:
Yeah, it was more that I wasn't certain that there wasn't a potential problem on Unix 🙂 I think from looking at the original opam client code that `Sys.command` and the `sh`-calling functions in `Unix` were used more... but we don't that at all anymore, so it's probably fine not to be conservative!

Reply:
The real [origin](https://github.com/ocaml/opam/commit/f3f6465b1fa406573917d46bc02f261f0906590f), it's there from the beginning ^^

Reply:
I think it's ok to remove it completely, processes handling has evolved since.